lib/sg.js

```diff
@@ -15,18 +15,14 @@
 'use strict';
 
 var _ = require('lodash');
-var _ec2;
-var _config;
-var _mode;
 
-
-
-var handleGroup = function(system, container, out, cb) {
+exports.handleSecurityGroup = function(mode, config, ec2, system, container, out, cb) {
   var sg = container.specific;
   var match = false;
-  var sgParams = { GroupIds: [container.nativeId] };
+  var filters = [];
+  filters.push({Name: 'tag:nscale-system', Values: [result.name + '-' + result.topology.name]});
 
-  _ec2.describeSecurityGroups(sgParams, function(err, sgroups) {
+  ec2.describeSecurityGroups(filters, function(err, sgroups) {
```

Comment:
From the documentation [1] does not look like you can pass an array of filters as param, you have to create a { Filters: filters } object

[1] http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/EC2.html#describeSecurityGroups-property